docs/CONTRIBUTING.md

```diff
@@ -15,6 +15,23 @@ You generally only need to submit a CLA once, so if you've already submitted one
 (even if it was for a different project), you probably don't need to do it
 again.
 
+## RAD Lab Repository
+
+We love and encourage your contributions to RAD Lab repo. Follow below steps to contribute on `features` and `bug fixes` for RAD Lab:
+
+1. **Fork** the public [RAD Lab repo](https://github.com/GoogleCloudPlatform/rad-lab)
+2. Create a `feature branch` from the `main` branch of your **forked** repo.
+
+    NOTE: Before you create the `feature branch` make sure that the `main` branch of your **forked** repo is in sync with the `main` branch of public RAD Lab repo.
+
+3. Once the development is completed, raise a PR to merge the `feature branch` (of **forked** repo) into [`staging`](https://github.com/GoogleCloudPlatform/rad-lab/tree/staging) branch of **RAD Lab** public repo.
+
+    NOTE: Make sure that there are **no conflicts** and most of the _testing is completed on the raised PR itself_.
```

Comment:
NOTE: Make sure that there are **no conflicts** and the _testing is completed on the raised PR itself_.